package/rpm/magnum-plugins.spec

```
@@ -0,0 +1,119 @@
+Name: magnum-plugins
+Version: 2020.06.1513.g72597410
+Release: 1
+Summary: Plugins for the Magnum C++11/C++14 graphics engine
+License: MIT
+Source: %{name}-%{version}.tar.gz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+Requires: magnum, DevIL, libpng, libjpeg-turbo, freetype, assimp, faad2-libs, glslang, spirv-tools-libs, libwebp, libspng, openexr-libs
+BuildRequires: cmake, git, gcc-c++, DevIL-devel, libpng-devel, libjpeg-turbo-devel, freetype-devel, assimp-devel, faad2-devel, harfbuzz-devel, glslang-devel, spirv-tools-devel, libwebp-devel, libspng-devel, openexr-devel
+Source1: https://github.com/BinomialLLC/basis_universal/archive/refs/tags/1.15_final.zip
```

Should work with this as well. This release optionally uses OpenCL, but the dependency list is massive enough already, so I personally wouldn't add it.

```suggestion
Source1: https://github.com/BinomialLLC/basis_universal/archive/refs/tags/v1_50_0_2.tar.gz
```

It also needs to be updated in the paths below.